tests/roles/pcp_cleanup/tasks/main.yaml

```diff
@@ -2,6 +2,16 @@
   ansible.builtin.shell: |
     {{ shell_header }}
     {{ oc_header }}
+    cd {{ install_yamls_path }}
+    # Delete the operator, but does not cleanup the service resources
+    make openstack_cleanup
```

Comment:
does this work ok? I've always done `openstack_deploy_cleanup` before `openstack_cleanup`

Reply:
if you remove the operators first then the below resource cleanup wont be complete as the operators won't do the cleanup tasks. So I think it is wrong.